samples/snippets/deletes/deletes_test.py

```diff
@@ -0,0 +1,144 @@
+    if table.exists():
+        table.delete()
+
+    table.create(column_families={"stats_summary": None, "cell_plan": None})
+
+    timestamp = datetime.datetime(2019, 5, 1)
+    timestamp_minus_hr = datetime.datetime(2019, 5, 1) - datetime.timedelta(hours=1)
+
+    row_keys = [
+        "phone#4c410523#20190501",
+        "phone#4c410523#20190502",
+        "phone#4c410523#20190505",
+        "phone#5c10102#20190501",
+        "phone#5c10102#20190502",
+    ]
+
+    rows = [table.direct_row(row_key) for row_key in row_keys]
+
+    rows[0].set_cell("stats_summary", "connected_cell", 1, timestamp)
+    rows[0].set_cell("stats_summary", "connected_wifi", 1, timestamp)
+    rows[0].set_cell("stats_summary", "os_build", "PQ2A.190405.003", timestamp)
+    rows[0].set_cell("cell_plan", "data_plan_01gb", "true", timestamp_minus_hr)
+    rows[0].set_cell("cell_plan", "data_plan_01gb", "false", timestamp)
+    rows[0].set_cell("cell_plan", "data_plan_05gb", "true", timestamp)
+    rows[1].set_cell("stats_summary", "connected_cell", 1, timestamp)
+    rows[1].set_cell("stats_summary", "connected_wifi", 1, timestamp)
+    rows[1].set_cell("stats_summary", "os_build", "PQ2A.190405.004", timestamp)
+    rows[1].set_cell("cell_plan", "data_plan_05gb", "true", timestamp)
+    rows[2].set_cell("stats_summary", "connected_cell", 0, timestamp)
+    rows[2].set_cell("stats_summary", "connected_wifi", 1, timestamp)
+    rows[2].set_cell("stats_summary", "os_build", "PQ2A.190406.000", timestamp)
+    rows[2].set_cell("cell_plan", "data_plan_05gb", "true", timestamp)
+    rows[3].set_cell("stats_summary", "connected_cell", 1, timestamp)
+    rows[3].set_cell("stats_summary", "connected_wifi", 1, timestamp)
+    rows[3].set_cell("stats_summary", "os_build", "PQ2A.190401.002", timestamp)
+    rows[3].set_cell("cell_plan", "data_plan_10gb", "true", timestamp)
+    rows[4].set_cell("stats_summary", "connected_cell", 1, timestamp)
+    rows[4].set_cell("stats_summary", "connected_wifi", 0, timestamp)
+    rows[4].set_cell("stats_summary", "os_build", "PQ2A.190406.000", timestamp)
+    rows[4].set_cell("cell_plan", "data_plan_10gb", "true", timestamp)
+
+    table.mutate_rows(rows)
+
+    # Ensure mutations have propagated.
+    row_set = RowSet()
+
+    for row_key in row_keys:
+        row_set.add_row_key(row_key)
+
+    fetched = list(table.read_rows(row_set=row_set))
+
+    while len(fetched) < len(rows):
+        time.sleep(5)
+        fetched = list(table.read_rows(row_set=row_set))
+
+    yield table_id
+
+
+def test_delete_from_column(capsys, snapshot, table_id):
+    deletes_snippets.delete_from_column_sample(PROJECT, BIGTABLE_INSTANCE, table_id)
+    out, _ = capsys.readouterr()
+    snapshot.assert_match(out)
+
+
+def test_delete_from_column_family(capsys, snapshot, table_id):
+    deletes_snippets.delete_from_column_family_sample(
+        PROJECT, BIGTABLE_INSTANCE, table_id
+    )
+    out, _ = capsys.readouterr()
+    snapshot.assert_match(out)
```

Comment:
Hmm. I see the `reads_test` doesn't do it that way though. It might be nice if all the samples were done this way, but I'm really curious about your thoughts.